intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/retrieval_agent.py

```diff
@@ -88,7 +88,7 @@ def append_localdb(self,
                        append_path, 
                        top_k=1, 
                        search_type="similarity_score_threshold", 
-                       search_kwargs={"score_threshold": 0.9, "k": 1}):
+                       search_kwargs={"score_threshold": 0.8, "k": 1}):
```

Comment:
why 0.8?

Reply:
Same like another comment, this is also for higher quality of retrieval